### DISCUSSION TITLE
Adding renovate randy manually

### DISCUSSION THREAD

**Discussion by sthumma3-dot** [renovatebot/renovate#19212] [Request Help]
Hi 
I recently started working on renovate randy and Without giving RRANDY write access I manually merged renovate.json into our repos and now I have given write access will that create any problem or still its works fine?

**Comment by rarkins** [comment]
What's randy?

**Reply by sthumma3-dot** [reply]
I mean the renovate bot

**Reply by sthumma3-dot** [reply]
@rarkins

**Reply by rarkins** [reply]
If it doesn't have write access then it will abort early in the repo. If it has write access then it will continue.

**Comment by sthumma3-dot** [comment]
@rarkins I gave write access after adding json will that be fine?

**Reply by sthumma3-dot** [reply]
So you mean It will not create a PR?
